src/dos/dos_locale_data.cpp

```diff
@@ -918,6 +1144,8 @@ const std::vector<KeyboardLayoutInfoEntry> LocaleData::KeyboardLayoutInfo = {
 	},
 	{
 		{ "yu" }, "Yugoslavian (QWERTZ)",
+		// Balkan countries seem to be mostly using the QWERTZ layouts
+		AutodetectionPriority::High,
```

Comment:
Man, going through all these layout must have been a lot of work. Nice job 🎉

Reply:
Not that much, actually. Matching the host keyboard layouts to guest ones was much more time consuming.